lib/pharos/phase_manager.rb

```diff
@@ -11,7 +11,8 @@ class PhaseManager
       Excon::Error,
       K8s::Error,
       Pharos::SSH::RemoteCommand::ExecError,
-      Errno::ECONNRESET
+      Errno::ECONNRESET,
+      Errno::ECONNREFUSED
```

Comment:
We need to retry these because hosts might be still booting when we execute `up`.